server/src/main/java/org/opensearch/rest/action/search/RestMultiSearchAction.java

```diff
@@ -113,8 +113,12 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
             }
         }
         return channel -> {
-            final RestCancellableNodeClient cancellableClient = new RestCancellableNodeClient(client, request.getHttpChannel());
-            cancellableClient.execute(MultiSearchAction.INSTANCE, multiSearchRequest, new RestToXContentListener<>(channel));
+            try { 
+                final RestCancellableNodeClient cancellableClient = new RestCancellableNodeClient(client, request.getHttpChannel());
+                cancellableClient.execute(MultiSearchAction.INSTANCE, multiSearchRequest, new RestToXContentListener<>(channel));
+            } finally {
+                cancellableClient.close();
```

Comment:
This leads to a compile error. It needs to use a try-with-resources block and not wrap it in a try-finally

```
/workspace/OpenSearch/server/src/main/java/org/opensearch/rest/action/search/RestMultiSearchAction.java:120: error: cannot find symbol
                cancellableClient.close();
                ^
  symbol:   variable cancellableClient
  location: class RestMultiSearchAction
```

Reply:
fixed in #8